src/qt/locale/bitcoin_zh.ts

```diff
@@ -430,8 +430,8 @@
         <translation>&amp;命令行选项</translation>
```

Comment:
mind to clarify what is "List of used sending address" Does it mean the history of sending order?